backoffice/backoffice/workflows/constants.py

```diff
@@ -29,15 +29,19 @@ class WorkflowType(models.TextChoices):
 
 DEFAULT_WORKFLOW_TYPE = WorkflowType.HEP_CREATE
 
-# author dags for each workflow type
-WORKFLOW_DAG = {
-    WorkflowType.HEP_CREATE: "",
-    WorkflowType.HEP_UPDATE: "",
-    WorkflowType.AUTHOR_CREATE: "author_create_initialization_dag",
-    WorkflowType.AUTHOR_UPDATE: "author_update_dag",
-}
-
 
 class ResolutionDags(models.TextChoices):
     accept = "accept", "author_create_approved_dag"
     reject = "reject", "author_create_rejected_dag"
+
+
+WORKFLOW_DAGS = {
```

Comment:
we can make them TextChoises, so we can easily access them instead of using indices

Reply:
I dont see how this can be implemented in this scenario,
the dict keys are already coming from the `workflow_type` textchoice.

An alternative I see so we don't index them would be to turn it into a dict of dicts
```

WORKFLOW_DAGS = {
    WorkflowType.HEP_CREATE: {},
    WorkflowType.HEP_UPDATE: {},
    WorkflowType.AUTHOR_CREATE: {
        "initialize": "author_create_initialization_dag",
        "approve": "author_create_approved_dag",
        "reject": "author_create_rejected_dag",
    },
    WorkflowType.AUTHOR_UPDATE: {"update":"author_update_dag"}
}
```
would this make sense?